pay-api/tests/unit/services/test_statement.py

```diff
@@ -628,7 +629,7 @@ def test_get_eft_statement_with_invoices(session):
                     'corp_type_code': 'CP',
                     'created_by': 'test',
                     'created_name': 'test name',
-                    'created_on': date_string_now,
+                    'created_on': get_local_formatted_date(invoice_2.created_on),
```

Comment:
Didn't have much choice with this one, it's formatted in local on purpose for the template